Source/TailwindTraders.Mobile/TailwindTraders.Mobile.Android/ThirdParties/Camera/Listeners/CameraSurfaceTextureListener.cs

```diff
@@ -28,7 +28,6 @@ public bool OnSurfaceTextureDestroyed(Android.Graphics.SurfaceTexture surface)
 
         public void OnSurfaceTextureSizeChanged(Android.Graphics.SurfaceTexture surface, int width, int height)
```

Comment:
Is this method any longer needed?

Reply:
Yes, imposed by TextureView.ISurfaceTextureListener